winforms/tests_backend/widgets/base.py

```diff
@@ -115,13 +115,16 @@ def assert_layout(self, size, position):
     async def press(self):
         self.native.OnClick(EventArgs.Empty)
 
-    async def type_character(self, char):
+    async def type_character(self, char, *, ctrl=False):
         try:
             key_code = KEY_CODES[char]
         except KeyError:
             assert len(char) == 1, char
             key_code = char
 
+        if ctrl:
+            key_code = "^" + key_code
```

Comment:
It turns out the Winforms ListView doesn't have a built-in select-all key combination, but I'll leave this here in case it's useful in the future.